pkg/plugins/pg/source/source.go

```diff
@@ -40,20 +39,17 @@ var (
 	// ErrInvalidURL is returned when the DB can't be connected to with the
 	// provided URL
 	ErrInvalidURL = cerrors.New("incorrect url")
+	// ErrEmptyConfig is returned when no config is provided
+	ErrEmptyConfig = cerrors.Errorf("must provide a plugin config")
```

Comment:
Nitpick, with a disclaimer of me not being a native English speaker :D When I hear "empty config" I think of a config that actually exists but has nothing in it. The comment and the error text though indicate there's no config at all.

Reply:
I think you're right, I updated it to `ErrNilConfig` which I think is more descriptive.